src/api/API.ts

```diff
@@ -30,17 +30,18 @@ export class API {
     return result;
   }
 
-  async createCustomer(customer: CustomerDraft): Promise<CustomerSignInResult> {
-    let result: CustomerSignInResult = {} as CustomerSignInResult;
+  async createCustomer(customer: CustomerDraft): Promise<{ data: CustomerSignInResult | undefined; error: string }> {
+    const errorMsg = '';
     try {
-      const { body } = await this.client.customers().post({ body: customer }).execute();
-      result = body;
-      if (result) {
-        alert('Congratulations! Your registration was successful. Welcome.');
-      }
+      const result = await this.client.customers().post({ body: customer }).execute();
+      return { data: result.body, error: 'error' };
     } catch (error) {
-      alert(error);
+      if (error instanceof Error) {
+        alert(
```

Comment:
I checked commercetools docs. Error object have the next format:

{
  "statusCode": 401,
  "message": "invalid_token",
  "errors": [
    {
      "code": "invalid_token",
      "message": "invalid_token"
    }
  ],
  "error": "invalid_token"
}

maybe it is better to use

`alert(error.massage);`
?

It will provide the reason why operation is failed